src/MatdaAIga.Generator/input/links/schema/links.yaml.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Profile Links Schema",
+  "description": "YAML schema for links page",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "title of document"
+    },
+    "links": {
+      "type": "array",
+      "description": "array of links",
+      "items": {
+        "type": "object",
+        "properties": {
+          "title": {
+            "type": "string",
+            "description": "title of link"
+          },
+          "url": {
+            "type": "string",
+            "description": "url of link",
+            "format": "uri",
+            "pattern": "^https?://"
+          },
+          "image_url": {
+            "type": "string",
+            "description": "url of image",
```

Comment:
`URL of the image used for the 'src' attribute of the 'img' tag. It must start with either '/images/' or 'https://'.`